types/FrameTypes.ts

```diff
@@ -124,3 +125,9 @@ export enum UpdateZIndexMethod {
     bringForward = 'bringForward',
     sendBackward = 'sendBackward',
 }
+
+export enum ShapeType {
+    circle = 'circle',
```

Comment:
Is it actually circle? At some point I think the implementation was more like 'oval', because on resize, it would try to fill the rectangle?

Reply:
It is indeed a responsive circle, should I call it "ellipse" or keep "circle" (it is if the bounding box is a square 😅)

Reply:
@Dvergar indeed, `ellipse` seems the correct mathematical terminology. Let's go for that!